src/lib/blueprint.ts

```diff
@@ -1,15 +1,90 @@
 import type { Openapi } from './openapi.js'
 
+interface Parameter {
+  name: string
+  isRequired: boolean
+  description: string
+}
+
+interface Response {
+  description: string
+}
+
+type Method = 'GET' | 'POST' | 'PUT' | 'DELETE' | 'PATCH'
+
+interface Endpoint {
+  name: string
+  path: string
+  method: Method[]
+  semanticMethod?: string
+  routeDescription?: string | undefined
+  parameters: Parameter[]
```

Comment:
We still need to figure out where to set this from, but for now we can default `isDeprecated` to false.

We can default deprecationMessage to `''` when `isDeprecated` is `false`.

```suggestion
  isDeprecated: boolean
  deprecationMessage: string
  parameters: Parameter[]
```

Reply:
The endpoint can be deprecated as well? Or did I misunderstand something?